src/Metadata/Property/Factory/CachedPropertyMetadataFactory.php

```diff
@@ -28,6 +28,7 @@
 
     private $cacheItemPool;
     private $decorated;
+    private $memoryCache = [];
```

Comment:
The word memory is just redundant and meaningless since all the variables are in memory anyway...